packages/components/src/components/event/eventHeroClosed/EventClosedHero.tsx

```diff
@@ -1,16 +1,25 @@
-import { useLocale } from '@events-helsinki/components';
 import { Button } from 'hds-react';
-import { useRouter } from 'next/router';
+import type { NextRouter } from 'next/router';
 import { useTranslation } from 'next-i18next';
 import React from 'react';
-import AppConfig from '../../app/AppConfig';
-import routerHelper from '../../app/routerHelper';
+import type { AppConfig, CmsRoutedAppHelper } from '../../../index';
+import type { AppLanguage } from '../../../types';
 import styles from './eventClosedHero.module.scss';
 
-const EventClosedHero: React.FC = () => {
+export interface EventCloseHeroProps {
+  appConfig: AppConfig;
+  router: NextRouter;
+  routerHelper: CmsRoutedAppHelper;
+  locale: AppLanguage;
+}
```

Comment:
Instaed of giving all these *HUGE* classes, tools and utils, the actual configuration values should be given.

Reply:
Otherwise the component needs much features that it does not even need, which makes it hard to test and mock. Also, the component rerenders everytime a prop changes, so the result can be lots of unneeded rerenders. Also, it could make the UI slow in performance.